third-party-major.json

```diff
@@ -67,8 +73,13 @@
       "allowedVersions": "!/^(3\\.42\\.0|4\\.21\\.0)$/"
     },
     {
-      "excludePackageNames": ["braid-design-system", "sku", "skuba"],
-      "excludePackagePatterns": ["^@?seek", "seek$"],
+      "matchPackageNames": [
+        "!braid-design-system",
+        "!sku",
+        "!skuba",
+        "!/^@?seek/",
+        "!/seek$/"
```

Comment:
Existing issues, but should we

- Enforce `@seek/` rather than `@seek`
- Drop the dubious `seek` suffix one and instead allowlist specific packages that we trust